### DISCUSSION TITLE
Purfview's Faster Whisper Batch

### DISCUSSION THREAD

**Discussion by Fetterbr61** [Purfview/whisper-standalone-win#247] [Q&A] [answered]
Hi.
I am new to this program but i like it until now :) Question.  I am using the program Subtitle Edit v4.0.5 with Batch Audio to Text Purfview's Faster Whisper, where can I add the Fix common errors or do i have to reload every single movie again to do this ?

**Comment by Purfview** [comment, marked as answer]
"Fix common errors" is SE's feature, so you should ask about it there. 
I think that is for OCR errors, I don't see how that is relevant to the whisper transcriptions.
